### ISSUE TITLE
404/401

### ISSUE THREAD

**Issue by schuppeste** [JanAckermann/owncloud-restore-trash#3]
Problem on start with right data and url https://www.mydoman.tld/owncloud:

Collection files to restore
ERROR: The requested URL returned error: 404 Not Found

**Comment by AlexAndBear** [comment]
What is the actual URI of your WebDAV? you can easily see this looking at the network tab(XHR) in your browser after logging in to your ownCloud

**Comment by schuppeste** [comment]
Thanks for your answer :) Vey Fast.

https://domain.tld/owncloud/remote.php/dav/files/username/

404:
File not found: trash-bin in 'root'

**Comment by tophneal** [comment]
I'm also getting a 401 error when trying to use this tool with either base url of our install, or the webDAV URL from the user settings in Files. Should this be run from on or off the server hosting Owncloud/Nextcloud?

**Comment by schuppeste** [comment]
I solved it another way..

i changed users password who deleted the Items, logged in as this user and checked  "all" Checkbox in Trash to Restore, Ajax do it for you time by time. you can check the Progression on File Count by Checked "Mark All" Preview.

**Comment by AlexAndBear** [comment]
@tophneal 401 means unauthorized, you need to provide the username and password of the user you want to restore the data

**Comment by tophneal** [comment]
Yes, and I did, using the flags as shown in the readme, and the shorter -u and -p options as described by the suggestion of the error output. I'll rerun it when I'm able and share a screenshot of how my command is typed out.
